.github/workflows/vale.yml

```
@@ -11,9 +11,10 @@ on:
 jobs:
   vale:
     name: linter
-    runs-on: ubuntu-latest
+    # workaround for https://github.com/errata-ai/vale-action/issues/128
+    runs-on: ubuntu-22.04
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v3
```

v4 should work just fine.
```suggestion
      - uses: actions/checkout@v4
```

---

OK, will update